paymentsheet/src/main/java/com/stripe/android/paymentsheet/state/LinkState.kt

```diff
@@ -10,9 +10,6 @@ internal data class LinkState(
     val loginState: LoginState,
 ) : Parcelable {
 
-    val isReadyForUse: Boolean
```

Comment:
Does this need to be part of this PR?

Reply:
The pull request removes its only usage, so I want to get rid of it too.